Migration.md

```diff
@@ -5,6 +5,12 @@ Deprecated code produces compile-time warnings. These warning serve as
 notification to users that their code should be upgraded. The next major
 release will remove the deprecated code.
 
+## Gazebo 9.15 to 9.16
+
+### Modifications
+
+Updated the version of TinyOBJLoader from 1.0.0 to 2.0.0rc8.
```

Comment:
I see that the API seems to be working after the bump in major version. Not sure if there is any behavior change or anything technical worth to mention here.

Reply:
Added a link to the changelog